modules/wallet/money.go

```diff
@@ -111,6 +113,62 @@ func (w *Wallet) SendSiacoins(amount types.Currency, dest types.UnlockHash) ([]t
 	return txnSet, nil
 }
 
+// SendSiacoinsMulti creates a transaction sending each amount in 'amounts' to
+// it's corresponding destination in 'dests'. The transaction is submitted to
+// the transaction pool and is also returned.
+func (w *Wallet) SendSiacoinsMulti(amounts []types.Currency, dests []types.UnlockHash) ([]types.Transaction, error) {
+	if err := w.tg.Add(); err != nil {
+		return nil, err
+	}
+	defer w.tg.Done()
+	if !w.unlocked {
+		w.log.Println("Attempt to send coins has failed - wallet is locked")
+		return nil, modules.ErrLockedWallet
+	}
+	if len(amounts) != len(dests) {
+		return nil, errors.New("number of amounts and destinations must match")
+	}
```

Comment:
need to check that the number of amounts is not larger than what can reasonably fit in an IsStandard transaction - 32kb. We should probably target 24kb or so, to give wiggleroom for weirdness (signatures may be large, or you may be spending tons of outputs, or something).